csrc/scheduler/transpose.cpp

```diff
@@ -36,10 +36,10 @@ class DomainMap : public pointwise_utils::DomainMap {
 
   TensorView* findReferenceFor(const std::vector<TensorView*>& group) const {
     TensorView* result = nullptr;
-    int max_dims = -1;
+    size_t max_dims = 0;
```

Comment:
I think it'd be safer to keep this a signed type as `nRootDims` may return 0 if we support 0-dim tensors.